build-llvm60.md

```diff
@@ -7,26 +7,49 @@ slug: build-llvm60
 
 {% include version_warning.md %}
 
-The current procedure for building KLEE with LLVM 6.0 (recommended) is outlined below. 
+The current procedure for building KLEE with LLVM 6.0 (recommended) is
```

Comment:
Your editor might be misconfigured: it introduces annoying line breaks in the middle of sentences (see below).